lib/index.js

```diff
@@ -80,9 +76,12 @@ module.exports = function (options) {
     tu.make_client(make_send, client_options, clientdone)
```

Comment:
^ This is the line that "breaks" when you send in args with "pins"... 

An alternative might be to not call this at all, maybe there is another way to hook make_send (below) into seneca?  Maybe copy/paste some logic from transport_utils.js